.circleci/ci/src/pipelines/tests/pipeline-full-release.spec.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as fs from 'fs';
+import { generateFullReleaseConfig } from '../pipeline-full-release';
+
+describe('Full release tests', () => {
+  it.each`
+    branch     | isDryRun | expectedResult
+    ${'4.2.x'} | ${true}  | ${'release-4-2-0-dry-run.yml'}
+    ${'4.2.x'} | ${false} | ${'release-4-2-0-no-dry-run.yml'}
+  `('should build full release config on $branch with dry run $isDryRun', ({ branch, isDryRun, expectedResult }) => {
```

Comment:
Is it possible that a release could be an alpha or that we may or may specify dockerTagAsLatest?

Reply:
As it has been tested in other tests files I didn't add it but I can add it here too yes 👍🏼

Reply:
Done